tests/components/knx/test_expose.py

```diff
@@ -1,16 +1,53 @@
 """Test knx expose."""
-from unittest.mock import AsyncMock, Mock
 
-from homeassistant.components.knx.expose import KNXExposeSensor
+from unittest.mock import AsyncMock, Mock, patch
 
+import pytest
 
-async def test_binary_expose(hass):
-    """Test that a binary expose sends only telegrams on state change."""
-    e_id = "fake.entity"
+from homeassistant.components.knx import (
+    CONF_KNX_EXPOSE,
+    CONFIG_SCHEMA as KNX_CONFIG_SCHEMA,
+    KNX_ADDRESS,
+)
+from homeassistant.components.knx.const import DOMAIN as KNX_DOMAIN
+from homeassistant.const import CONF_ATTRIBUTE, CONF_ENTITY_ID, CONF_TYPE
+from homeassistant.setup import async_setup_component
+
+
+async def setup_knx_integration(hass, knx_mock, config={}):
```

Comment:
Create the default empty dict in the function body instead and default to None in the signature.